src/core/atomicdex/api/mm2/rpc2.task.enable_z_coin.init.hpp

```diff
@@ -34,7 +34,7 @@ namespace atomic_dex::mm2
         std::vector<atomic_dex::electrum_server>  servers;
         std::vector<std::string>                  z_urls;
         CoinType                                  coin_type;
-        int                                       sync_date{1672531200};   // Jan 1st 2023
+        int                                       sync_height{0};
```

Comment:
is 0 default here? or it's just there?

Reply:
In case there is no value in coins_config.json, it will default to 0. Its not defined for ZOMBIE for example, but there is no rush as it is a small chain at the moment.